docs/features/private-ip.md

```diff
@@ -1,30 +1,53 @@
 # Using Private IP for internal routing
 
-## Pre-requisites
-* Application Gateway with a [Private IP configuration](https://docs.microsoft.com/en-us/azure/application-gateway/configure-application-gateway-with-private-frontend-ip)
+This feature allows to expose the ingress endpoint within the `Virtual Network` using a private IP.
 
-This feature allows to expose the ingress endpoint within the `Virtual Network`.
-To configure the controller to use Private IP for routing, modify the `helm` config by adding `usePrivateIP: true`
+**Pre-requisites**
+Application Gateway with a [Private IP configuration](https://docs.microsoft.com/en-us/azure/application-gateway/configure-application-gateway-with-private-frontend-ip)
 
-## Example
+There are two ways to configure the controller to use Private IP for ingress,
+
+## Assign to a particular ingress
+To expose a particular ingress over Private IP, use annotation [`use-private-ip`](../annotations.md#use-private-ip) in Ingress.
+
+### Usage
+```yaml
+appgw.ingress.kubernetes.io/use-private-ip: "true"
+```
+
+If Application Gateway doesn't have a private IP, then Ingresses with this annotation set to `true` will be pruned. This will be indicated in the ingress event and AGIC pod log.
```

Comment:
I'd recommend we simplify this sentence to - `App Gateways without a Private IP and annotated with … will be ignored`